pkg/controller/servicebindingrequest/annotations/attribute.go

```diff
@@ -0,0 +1,53 @@
+package annotations
+
+import (
+	"github.com/redhat-developer/service-binding-operator/pkg/controller/servicebindingrequest/nested"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+const AttributeValue = "binding:env:attribute"
+
+// AttributeHandler handles "binding:env:attribute" annotations.
+type AttributeHandler struct {
+	// inputPath is the path that should be extracted from the resource. Required.
+	inputPath string
+	// outputPath is the path the extracted data should be placed under in the
+	// resulting unstructured object in Handler. Required.
+	outputPath string
+	// resource is the unstructured object to extract data using inputPath. Required.
+	resource unstructured.Unstructured
+}
+
+// Handle returns a unstructured object according to the "binding:env:attribute"
+// annotation strategy.
+func (h *AttributeHandler) Handle() (Result, error) {
+	val, _, err := nested.GetValue(h.resource.Object, h.inputPath, h.outputPath)
```

Comment:
Why are we ignoring the boolean value here

Reply:
I see the function `nested.GetValue` is used in two files and both of them ignore the boolean return value.

If we don't need the boolean reurn value, shouldn't we remove it from the function iteself?

```
➜  service-binding-operator git:(streamline-annotation-2) grep -rnI "nested.GetValue" .                                        
./pkg/controller/servicebindingrequest/service.go:207:                  val, _, err := nested.GetValue(obj.Object, br.inputPath, br.outputPath)
./pkg/controller/servicebindingrequest/annotations/attribute.go:24:     val, _, err := nested.GetValue(h.resource.Object, h.inputPath, h.outputPath)
➜  service-binding-operator git:(streamline-annotation-2) 
```

Reply:
It is ignored because the semantics for error and whether the value exist in the given object are same in those methods, but not necessarily always going to be the same for every caller.

Reply:
Please submit a PR if you'd like to have that changed.